SystemTest/main.go

```
@@ -1,48 +1,228 @@
+	if err != nil {
+		return err
+	}
+	if do.StatusCode != http.StatusOK {
+		return fmt.Errorf("status code was [%v]", do.StatusCode)
+	}
+	return nil
+}
+
+func (s *ServiceClient) GetAllItems() (*GetAllItemsResponse, error) {
+	///product/items
+	url := createUrl(fmt.Sprintf("product/items"))
```

golangci-lint (1.62.2)

S1039: unnecessary use of fmt.Sprintf

(gosimple)